src/test/java/com/revature/servicestests/SearchServiceTest.java

```diff
@@ -324,12 +327,14 @@ void searchServiceBadFormat()
 		String title = content.getTitle();
 		String format = content.getFormat();
 		List<Integer> mlist = new ArrayList<Integer>();
+		List<String> tempArr = new ArrayList<String>();
 		mlist.add(module1.getId());
 		mlist.add(module2.getId());
 
 		String badformat = "Document";
-		Set<Content> filtered = ss.filter(title, format, mlist);
-		filtered = ss.filter(title, badformat, mlist);
+		//tempArr IS TEMPORARY TO SILENCE ERRORS
```

Comment:
delete comment

Reply:
Comment necessary to indicate purpose of temporary variable